### DISCUSSION TITLE
Meeting, not no meet up but planning 2022-03-10

### DISCUSSION THREAD

**Discussion by jongfeel** [hongsam14/myCPU#39] [General]
계획이 없이 진행되다 보니 진척이 안되는 것 같기도 하고 그래서 계획을 세우고 시작하는 걸로 해보겠습니다.

우선 뭘 해야 할지에 대한 내용은 정한 거 같습니다.

- AND 게이트 바탕으로 해서 다른 게이트 작업 진행

그러면 아래 질문에 대해 신중히 생각해 보시고 답변을 한 후에 계획을 세워서 issue에 적어 보시죠.

1. 다음 미팅 날인 2022-03-25 까지 총 몇 시간을 투자할 수 있는지 계산 후 도출 (힌트, 이미 약속이 되어 있거나 안되는 날 제외)
2. 다음에 한 번 작업할 때 투자할 수 있는 시간과 날짜 정하기 (예, 3월 12일, 오후 1시 부터 4시간)
3. 그 시간 만큼 할 수 있는 작업을 issue에 등록하기 (예, OR gate prefab 작업)
4. 3의 작업이 완료 되면 2, 3을 반복 1에서 정한 총 투입 시간과 사용한 시간을 보고 얼마만큼 더 빨리 혹은 천천히 진행해도 되는지 판단
5. 1에서 정한 시간을 모두 소비할 수 있도록 진행, 단 작업이 완료되지 않아도 됨.
6. 1에서 정한 시간을 초과할 수도 있고 미달일 수도 있음 (대부분 미달) => 미달이라면 다음 계획은 미달된 시간 만큼 계획을 잡아야 함

**Comment by hongsam14** [comment]
여기에 일단 적고 이슈에 옮기겠습니다. (메모) 일단 비는 시간을 나열하면, (학교 과제 하는 시간을 제외하지 않은 시간)

3월 10일 21 : 00 ~ 23 : 00
3월 11일 20 : 00 ~ 22 : 00
3월 14일 20 : 00 ~ 23 : 00
3월 15일 21 : 00 ~ 23 : 00
3월 16일 18 : 00 ~ 23 : 00
3월 17일 14 : 00 ~ 23 : 00
3월 18일 18 : 00 ~ 23 : 00

이 다음주부턴 대학교 대면 수업이라 대면 일정이 나오면 잉여 시간을 알 것 같습니다.

**Comment by hongsam14** [comment]
병원에 입원해서 11일 계획은 12일로 미룰 예정
